docs/zh-Hans/Modules/Docs.md

```diff
@@ -176,37 +162,46 @@ ABP框架的[文档](docs.abp.io)也是使用的此模块.
 
 #### 4.1- Entity Framework 集成
 
-如果你选择了Entity Framework 做为数据库供应者,你需要在DbContext中配置文档模块. 做以下操作;
+如果你选择了Entity Framework 做为数据库供应者,你需要配置文档模块. 做以下操作;
 
-* 打开 `MyProjectDbContext.cs` 并且添加 `modelBuilder.ConfigureDocs()` 到 `OnModelCreating()` 方法中
+* 打开 `QaDocDbContextModelCreatingExtensions.cs` 并且添加 `modelBuilder.ConfigureDocs()` 到 `ConfigureMyProject()` 方法中
 
   ```csharp
-  [ConnectionStringName("Default")]
-  public class MyProjectDbContext : AbpDbContext<MyProjectDbContext>
+  public static class QaDocDbContextModelCreatingExtensions
   {
-      public MyProjectDbContext(DbContextOptions<MyProjectDbContext> options)
-          : base(options)
+      public static void ConfigureMyProject(this ModelBuilder builder)
       {
-  
+          Check.NotNull(builder, nameof(builder));
+
+          /* Configure your own tables/entities inside here */
+
+          //builder.Entity<YourEntity>(b =>
+          //{
+          //    b.ToTable(QaDocConsts.DbTablePrefix + "YourEntities", QaDocConsts.DbSchema);
+
+          //    //...
+          //});
+
+          builder.ConfigureDocs();
```

Comment:
I think the `ConfigureDocs `method should be located in the `XXXMigrationsDbContext `class. Right?